stock_no_negative/models/product_template.py

```diff
@@ -19,13 +19,20 @@
 #
 ##############################################################################
 from openerp import fields, models
+from openerp.tools import config
 
 
 class ProductTemplate(models.Model):
     _inherit = 'product.template'
 
+    # for not to break other tests
+    def _get_default_no_negative_policy(self):
+        return not (config['test_enable'] and
+                    not self.env.context.get('check_no_negative'))
+
     check_no_negative = fields.Boolean(
         'Check no negative',
+        default=_get_default_no_negative_policy,
```

Comment:
Although I agree with [@pedrobaeza comment](https://github.com/OCA/stock-logistics-workflow/pull/258#issuecomment-246260658) which would remove this entirely